content/sensu-go/5.20/release-notes.md

```diff
@@ -61,6 +62,26 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 5.20.2 release notes
+
+**May 26, 2020** &mdash; The latest release of Sensu Go, version 5.20.2, is now available for download.
+
+This patch release adds username to the API request log to help operators with troubleshooting and user activity reporting, as well as validation for subjects in role-based access control (RBAC) role binding and cluster role binding. Release 5.20.2 also temporariliy disables process discovery so we can investigate and resolve an increase in event payload size.
```

Comment:
> so we can investigate and resolve an increase in event payload size.

Maybe we should instead mention the performance impact of this feature on the backend

Reply:
👍 what do you think of this change?
```
Release 5.20.2 also temporarily disables process discovery so we can investigate and resolve its performance impact on the backend (increased CPU and memory usage).
```